src/Ch1/BMIDemo.java

```diff
@@ -0,0 +1,28 @@
+//Program to Compute Body Mass Index
+
+/**
+ * @Author Naresh
+ * @Year 2018
+ * @Version 1.1
+ */
+
+package Ch1;

```

Comment:
package folder names must follow lower case letters. I think I did not tell this during the training, but the package names must be lowercase.